### PR TITLE
Added locking section

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/php-task/php-task/pull/35

### CODE REVIEW

File: locking.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Locking
+=======
+To avoid concurrency issue the library is able to lock other executions which
```
Comment (danrot) [inline locking.rst]: `issues, the library`

File: locking.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+would run at the same time. This could happen if multiple worker (e.g. cronjobs)
```
Comment (danrot) [inline locking.rst]: `workers`

File: symfony.rst [hunk 3]
```diff
@@ -79,23 +79,34 @@ supported.
 After addition or changing in the config you have to run the command again
 to be sure that the task-table will be updated.
 
+Locking
+-------
+Locking is used to avoid concurrency problems when multiple task-runner runs at
```
Comment (danrot) [inline symfony.rst]: `task-runners run at`

File: symfony.rst [hunk 4]
```diff
@@ -79,23 +79,34 @@ supported.
...
-                clear:                true
+                clear:            true
         run:
             mode:                 'off' # One of "off"; "listener"
-        system_tasks:
-            enabled:              true
-            handler_class:        ~
-            workload:             null
-            cron_expression:      ~
+        locking:
```
Comment (danrot) [inline symfony.rst]: Does locking replace `system_tasks`?

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by wachterjohannes** [comment]
@danrot ping

**Pull request merged**